### PR TITLE
Segment component + minor UI change in global h1 font size

### PR DESCRIPTION
**Pull request by eevanwong**

### CODE REVIEW

File: components/charts/doughnut.js [hunk 1]
```diff
@@ -35,9 +35,9 @@ export default function DoughnutChart(props) {
 
   return (
     <div>
-      <div className="header">
+      {/* <div className="header">
```
Comment (NirmalHegde) [inline components/charts/doughnut.js:38]: Are these comments going to be needed in the future? If not they should be removed

File: components/charts/hsAverage.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import { db } from "../../firebaseConfig";
+import { Line } from "react-chartjs-2";
+
+const backup = db.collection("1A Data").doc("HS Average");
+const dataVal = [];
+const dataLabel = [];
+const dataColor = [];
+
+const observer = backup.onSnapshot(
```
Comment (NirmalHegde) [inline components/charts/hsAverage.js:9]: discussion: should logic for components be added in a separate file and passed in here when necessary?

File: components/charts/pieChart.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import { Pie } from "react-chartjs-2";
+
+export default function PieChart(props) {
+  const [data, setData] = useState({
+    val: [],
+    label: [],
+    color: [],
+  });
+
+  db.collection("1A Data")
```
Comment (NirmalHegde) [inline components/charts/pieChart.js:12]: same comment as other one about logic in diff file.

File: components/charts/pieChart.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+          options={{
+            responsive: true,
+            maintainAspectRatio: false,
+            legend: { display: false },
+            title: {
+              display: true,
+              text: data.title,
+            },
+          }}
+          height="450px"
```
Comment (NirmalHegde) [inline components/charts/pieChart.js:64]: charts should be dynamic to accomodate mobile viewing

File: styles/globals.scss [hunk 5]
```diff
@@ -31,7 +31,7 @@ body {
 
 h1 {
   font-weight: 600;
-  font-size: 42px;
```
Comment (NirmalHegde) [inline styles/globals.scss:34]: font should be dynamic for mobile viewing

**Review by NirmalHegde — CHANGES_REQUESTED** [review]
pls see comments before merging.

File: components/charts/doughnut.js (hunk 1 above)
Comment (eevanwong) [reply components/charts/doughnut.js:38]: they might be, they are headings but I was unsure if we should keep them or not as we have actual segment headings instead.

File: styles/globals.scss (hunk 5 above)
Comment (eevanwong) [reply styles/globals.scss:34]: we have css that changes fontSize depending on how wide the screen is already

File: components/charts/hsAverage.js (hunk 2 above)
Comment (eevanwong) [reply components/charts/hsAverage.js:9]: yeah, but this is def to be discussed later, we haven't determined atm the specifics

File: components/charts/pieChart.js (hunk 3 above)
Comment (eevanwong) [reply components/charts/pieChart.js:12]: same response as other

**Review by parminpatel28 — COMMENTED** [review]
You need to rebase with develop branch.

**Review by NirmalHegde — APPROVED** [review]

File: styles/globals.scss [hunk 6]
```diff
@@ -31,7 +31,7 @@ body {
...
+  font-size: 36px;
```
Comment (shivam-sh) [inline styles/globals.scss:34]: Was there a reason this change was made? 36 is awfully close to h2 and differs from the design guidelines we have in the figma

Comment (shivam-sh) [reply styles/globals.scss:34]: I’d recommend local CSS changes rather than local ones if this fixes a specific case

Comment (eevanwong) [reply styles/globals.scss:34]: I forgot to change that back, I remember it was because I had thought the title of each overall section was too small .

**Comment by eevanwong** [comment]
I change one thing and the test fail :/

**Review by parminpatel28 — APPROVED** [review]

**Pull request merged**